components/Dialogs/README.md

```diff
@@ -223,11 +248,89 @@ MDCAlertControllerThemer.applyScheme(scheme, to: alertController)
 // Step 1: Import the DialogThemer extension
 #import "MaterialDialogs+DialogThemer.h"
 
-// Step 2: Create or get a alert scheme
-MDCAlertScheme *alertScheme = [[MDCAlertScheme alloc] init];
+// Step 2: Create or get a Material container scheme
+MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+
+// Step 3 (optional): Customize the default theme by providing custom color or typography schemes:
+scheme.colorScheme = myColorScheme
+scheme.typographyScheme = myTypographyScheme
+
+// Step 4: Use the Material alert themer to theme an MDCAlertController instance
+[alertController applyThemeWithScheme:scheme];
+
+// Step 4: Alternatively, Use Material dialog presentation themer to theme your UIViewController instance:
+[myDialogViewController.mdc_dialogPresentationController applyThemeWithScheme: scheme];
```

Comment:
Nit/formatting: There is an extra space between `applyThemeWithScheme:` and `scheme`.